bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceAction.java

```diff
@@ -54,6 +58,37 @@
  */
 public class FindReplaceAction extends ResourceAction implements IUpdate {
 
+	private static final String INSTANCE_SCOPE_NODE_NAME = "org.eclipse.ui.editors"; //$NON-NLS-1$
+
+	private static final String USE_FIND_REPLACE_OVERLAY = "useFindReplaceOverlay"; //$NON-NLS-1$
+
+	private static final String FIND_REPLACE_OVERLAY_AT_BOTTOM = "findReplaceOverlayAtBottom"; //$NON-NLS-1$
+
+	private boolean shouldUseOverlay() {
+		IEclipsePreferences preferences = InstanceScope.INSTANCE.getNode(INSTANCE_SCOPE_NODE_NAME);
+		boolean overlayPreference = preferences.getBoolean(USE_FIND_REPLACE_OVERLAY, false);
+		return overlayPreference && fWorkbenchPart instanceof StatusTextEditor;
+	}
+
+	private static boolean shouldPositionOverlayOnTop() {
+		IEclipsePreferences preferences = InstanceScope.INSTANCE.getNode(INSTANCE_SCOPE_NODE_NAME);
+		return !preferences.getBoolean(FIND_REPLACE_OVERLAY_AT_BOTTOM, false);
+	}
+
+	private IPreferenceChangeListener overlayDialogPreferenceListener = new IPreferenceChangeListener() {
+
+		@Override
+		public void preferenceChange(PreferenceChangeEvent event) {
+			System.out.println(event.getKey());
```

Comment:
Please remove this

Reply:
https://github.com/eclipse-platform/eclipse.platform.ui/commit/2f9136a25069244769073ee77ad5be15b5dadeee